### ISSUE TITLE
'THEN' statement missing

### ISSUE THREAD

**Issue by neoshin0** [Fuminides/ex-fuzzy#13]
The code i wrote from your example:

import ex_fuzzy.evolutionary_fit as evf
import ex_fuzzy.fuzzy_sets as fs
import ex_fuzzy.utils as utils
import ex_fuzzy.eval_tools as eval_tools
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split
import pandas as pd

# Load the Iris dataset
iris = load_iris()
X = pd.DataFrame(iris.data, columns=iris.feature_names)
y = iris.target

# Split into training and testing sets
X_train, X_test, y_train, y_test = train_test_split(
    X, y, test_size=0.33, random_state=42
)

# Create linguistic variables (fuzzy partitions) from the data
linguistic_variables = utils.construct_partitions(X_train, fs.FUZZY_SETS.t1)

# Create a fuzzy classifier
classifier = evf.BaseFuzzyRulesClassifier(
    nRules=15,                            # Number of rules to evolve
    nAnts=4,                              # Maximum antecedents per rule
    linguistic_variables=linguistic_variables,  # Pre-computed fuzzy variables
    fuzzy_type=fs.FUZZY_SETS.t1,         # Type-1 fuzzy sets
    verbose=True                          # Show training progress
)

# Train the classifier
classifier.fit(X_train, y_train, n_gen=30, pop_size=50)

# Make predictions
y_pred = classifier.predict(X_test)

# Calculate accuracy manually since score method requires X input too
from sklearn.metrics import accuracy_score
accuracy = accuracy_score(y_test, y_pred)
print(f"Accuracy: {accuracy:.3f}")

# Print the learned rules
rule_text = classifier.print_rules(return_rules=True)
print(rule_text)

# Use comprehensive evaluation
evaluation_report = eval_tools.eval_fuzzy_model(
    classifier, X_train, y_train, X_test, y_test,
    plot_rules=True,      # Show rule structure
    print_rules=True,     # Print rules in readable format
    plot_partitions=True, # Show fuzzy partitions
    return_rules=True     # Return rule text
)

The result gives this one:

IF sepal length (cm) IS Low AND sepal width (cm) IS Low AND petal width (cm) IS Low WITH DS 0.00279, ACC 1.0
IF sepal width (cm) IS High AND petal length (cm) IS Low WITH DS 0.06472800000000001, ACC 1.0
IF petal length (cm) IS Low AND petal width (cm) IS Low WITH DS 0.0827948, ACC 1.0

while your actual results gives this one from your example ( reference: https://fuminides.github.io/ex-fuzzy/getting-started.html ):

Rule 1: IF sepal length (cm) is Low AND petal width (cm) is Low THEN setosa (DS: 0.85)
Rule 2: IF petal length (cm) is High AND petal width (cm) is High THEN virginica (DS: 0.92)
Rule 3: IF petal length (cm) is Medium AND sepal width (cm) is High THEN versicolor (DS: 0.78)

I need to know why I can't have the 'THEN' results from the code above.

**Comment by Fuminides** [comment]
Hi! For the standard classifiers that we use, the text format of the rules does not include the THEN clause because it would result in a lot of redundant text. To know which class correspond to each group, you have the class name before every block of rules, which would look for example like this:

Rules for consequent: 0
----------------
